spec/holdings_deleter_spec.rb

```diff
@@ -0,0 +1,131 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+require_relative "../bin/holdings_deleter"
+
+RSpec.describe HoldingsDeleter do
+  def neu(*args)
```

Comment:
FWIW the general preference for rspec style seems to be to use `described_class.new` for this case (to avoid repeating the class name and to make it slightly easier if you want to change the class name) and not to worry too much about the extra verbosity. Another alternative if you don't like `described_class` might be something like:

```ruby
let(:hd) { HoldingsDeleter }

it "does something" do
  hd.new(...)
end
```

Reply:
Oh I like that 2nd version.